shopfloor_mobile_base_auth_api_key/static/wms/src/login.js

```diff
@@ -21,7 +21,32 @@ export class ApiKeyAuthHandler extends AuthHandlerMixin {
             },
         };
     }
+
+    on_login($root, evt, data) {
+        evt.preventDefault();
+        // Call odoo application load => set the result in the local storage in json
+        // $root.apikey = data.apikey;
+        $root.apikey = data.apikey;
+        const def = $.Deferred();
+        return $root
+            ._loadConfig()
+            .then(function () {
+                if (!$root.authenticated) {
+                    return def.reject("screen.login.error.api_key_invalid");
+                }
+                return def.resolve();
+            })
+            .catch(function (error) {
+                return def.reject("screen.login.error.api_key_invalid");
```

Comment:
this is wrong: you are returning the translation key, not the translation itself
```suggestion
                return def.reject($root.$("screen.login.error.api_key_invalid"));
```